Tests/ChainsafeRPCTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿using NUnit.Framework;
+using Web3Unity.Scripts.Library.Ethers.Providers;
+
+namespace Tests
+{
+    [TestFixture]
+    public class ChainSafeRpcTests
+    {
+        private JsonRpcProvider _provider;
+        [SetUp]
+        public void SetUp()
+        {
+            _provider = new JsonRpcProvider("https://goerli.infura.io/v3/904006115c764661965dc0909e5ed473");
```

Comment:
if there is a better way to have the provider URL stored in other place, let me know

Reply:
You could create a singleton object that references the provider instance.